src/martok/Martok.ts

```diff
@@ -18,6 +18,12 @@ import { AsyncLocalStorage } from "async_hooks";
 import { TypeReplacer } from "./processing/TypeReplacer";
 import { processSnakeCase } from "./processing/SnakeCase";
 import { processOldNames } from "./processing/SanitizeNames";
+import {
```

Comment:
I wonder if we can compartmentalize the VFS dependencies into their own class? It seems like we're violating SRP (even more than we already were) by adding this extra logic in directly to this class.

Reply:
Good call. Should I pull all the VFS related functionality out into a `TsCompiler` class? And then make that call a member variable on the martok object?

Reply:
Yeah I think that works, or maybe `Recompiler` or just `VsCode` haha

Reply:
Or `Flattener`